hooks/useDialog.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react'
+import useIsMobile from './useIsMobile'
+
+const useDialog = () => {
+  const isMobile = useIsMobile()
+  const [isDialogOpen, setIsDialogOpen] = useState(false)
+  const [isVisibleToolTip, setIsVisibleTooltip] = useState(false)
+  const [tooltipX, setTooltipX] = useState(0)
+  const [tooltipY, setTooltipY] = useState(0)
+
+  const showDlg = () => {
```

Comment:
simpler + less `abbreviations`
```suggestion
  const show = () => {
```